### DISCUSSION TITLE
Collapsable nodes in the animation track editor

### DISCUSSION THREAD

**Discussion by shiv808** [godotengine/godot-proposals#8655] [Editor]
Hello, I am currently somewhat new to using Godot, I think it would be very helpful if it were possible to collapse nodes with animations on the animation track. 

![Screenshot 2023-12-14 122718](https://github.com/godotengine/godot-proposals/assets/54805593/30c7684d-60a1-429c-bf3b-0f8f05d5b34c)

I am working with Maximo base animations, and adding other animation properties to other nodes, and they show all the way at the bottom, I believe having the feature to collapse the skeleton3d node would help a ton, as i have to manually scroll down to the other properties I am editing)

thanks for reading.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/10276
